### PR TITLE
OL-10845 fix py2 syntax error and raise exception for empty values

### PR DESCRIPTION
**Pull request by mariusvigariu**

### CODE REVIEW

File: lib/ansible/module_utils/tacp_ansible/tacp_utils.py [hunk 1]
```diff
@@ -118,15 +118,16 @@ def get_filters_query_string(self, **kws):
 
         for k, v in kws.items():
             if isinstance(v, (list, tuple)):
-                op, *value = v
+                op, value = v[0], v[1:]
                 if op not in allowed:
                     raise Exception('Invalid operator "{}". '
                                     'Allowed: {}'.format(op, allowed))
+                if len(value) == 0:
+                    raise Exception('Must provide at least one value')
```
Comment (patrickjennings) [inline lib/ansible/module_utils/tacp_ansible/tacp_utils.py:126]: You could include the field and op in the exception message.

**Review by patrickjennings — APPROVED** [review]
Small nit but otherwise look good to me!

File: lib/ansible/module_utils/tacp_ansible/tacp_utils.py [hunk 2]
```diff
@@ -118,15 +118,16 @@ def get_filters_query_string(self, **kws):
...
 
                 if op in ('=in=', '=out='):
-                    if len(value) > 0:
-                        value = ','.join(map(str, value))
-                        value = '({})'.format(value)
+                    value = ','.join(map(str, value))
+                    value = '({})'.format(value)
```
Comment (xmadsen) [inline lib/ansible/module_utils/tacp_ansible/tacp_utils.py:130]: I ran into an issue with filtering templates if there was a space in the template's name - the value should have "" around it in the filter to include the whole text and not cut off at a space.

Comment (mariusvigariu) [reply lib/ansible/module_utils/tacp_ansible/tacp_utils.py:130]: Yea I remember something similar, that's probably the reason why I initially had the third placeholder wrapped in double quotes on line 136 below.

Do you still have an example to reproduce this? If yes I suggest you create a bug and take it separately.

Let's merge this if you guys dont see any issues with OL-10845 itself

**Pull request merged**